build/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM maven:3.8.6-eclipse-temurin-17
+FROM maven:3.9.11-eclipse-temurin-17
 
 RUN addgroup --gid 1000 build && adduser --uid 1000 --gid 1000 --disabled-password --gecos "" build
```

Comment:
hi @ivy-rew 

i think i saw you delete this line in some of our repos? this PR build seems to be failing because of this line.
do we want to delete this line on this branch as well?

Reply:
Mhh I think we will rather discard this update, but @alexsuter knows best.
The thing is that we would raise the Maven version beyond that what was available at release time of LTS10. 
So I think we rather stick to the version we have.

Neverthless, it's true that the user/gid 1000 is already taken in the new base image, so there's no need to add it on our own terms.